block/block.go

```diff
@@ -257,7 +257,7 @@ func (b *Block) GetInfo() ([]byte, error) {
 	info := &blockInfo{
 		Header:       unmarshaledHeader,
 		Transactions: make([]interface{}, 0),
-		Size:         b.ToMsgBlock().Size(),
+		Size:         proto.Size(b.ToMsgBlock()),
```

Comment:
why chosing `proto.Size` instead? And they both return the same value in this case right?

Reply:
Google's pb only has this API, it returns the same result